### PR TITLE
Update to MUX 2.1.190131001-prerelease

### PR DESCRIPTION
**Pull request by jevansaks**

### CODE REVIEW

File: XamlControlsGallery/XamlControlsGallery.csproj [hunk 1]
```diff
@@ -1,1069 +1,1069 @@
...
+    <Compile Include="ControlPages\AcrylicPage.xaml.cs">
+      <DependentUpon>AcrylicPage.xaml</DependentUpon>
+    </Compile>
+    <Compile Include="ControlPages\AppBarButtonPage.xaml.cs">
+      <DependentUpon>AppBarButtonPage.xaml</DependentUpon>
+    </Compile>
+    <Compile Include="ControlPages\AppBarPage.xaml.cs">
+      <DependentUpon>AppBarPage.xaml</DependentUpon>
+    </Compile>
+    <Compile Inclu
```
[the remark this answers is not in the document]
Comment (jevansaks) [reply XamlControlsGallery/XamlControlsGallery.csproj]: Looks like some tabs are changing to spaces and possibly some line endings towards the end changed. But i don't know why the diff is showing all lines different.

[the remark this answers is not in the document]
Comment (jevansaks) [reply XamlControlsGallery/XamlControlsGallery.csproj]: If you diff ignoring whitespace you'll see the important difference. I only edited via Visual Studio so presumably this is what the next unsuspecting person would encounter as well?

[the remark this answers is not in the document]
Comment (stmoy) [reply XamlControlsGallery/XamlControlsGallery.csproj]: How curious. Did you update this file by hand? When I've rev'd the package in the past using NuGet manager, it just modifies that one line. This is the first time that the whole file updated so I'm worried about the opposite: if I accept this change, the next unsuspecting person (probably me 😅) will get hit by this and update the whole file back.

[the remark this answers is not in the document]
Comment (jevansaks) [reply XamlControlsGallery/XamlControlsGallery.csproj]: Could you checkout the branch and retarget the nuget package and see what result you get? Maybe it's some configuration issue on my machine.

Oh I also was using VS2019, I bet that confused something...

[the remark this answers is not in the document]
Comment (stmoy) [reply XamlControlsGallery/XamlControlsGallery.csproj]: Oddly enough, the only difference that VS sees is the one line that has changed. I'll approve this PR and hope for the best for next time.

**Pull request merged**